ansible/library/port_alias.py

```diff
@@ -134,8 +142,14 @@ def get_portmap(self, asic_id=None):
                         aliasmap[alias] = name
                         if (speed_index != -1) and (len(mapping) > speed_index):
                             portspeed[alias] = mapping[speed_index]
+                        if (asic_name_index != -1) and (len(mapping) > asic_name_index):
+                            asicifname = mapping[asic_name_index]
+                            asicifnames.append(asicifname)
+                    if (asic_name_index != -1) and (len(mapping) > asic_name_index):
+                        asicname = mapping[asic_name_index]
+                        asicnames.append(asicname)
 
```

Comment:
the asicnames list will have both internal and external interfaces ?
Can you elaborate what is the difference between asicnames and asicifnames ?

Reply:
Changed to front_panel_asic_ifnames and asic_if_names to make it clear, updated comment in port_alias.py